src/test/java/step2/StringCalculateTest.java

```diff
@@ -0,0 +1,87 @@
+package step2;
+
+
+import Calculator.StringCalculator;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class StringCalculateTest {
+    //Calculator calculator;
+
+    @Test
+    @DisplayName("정상적인 덧셈 테스트")
+    void AddTestFunction() {
+        StringCalculator calculator = new StringCalculator();
+        double result = calculator.calculate("1 + 2 + 3 + 4");
+        assertThat(result).isEqualTo(10);
+    }
+    @Test
```

Comment:
StringCalculate 에서 메서드 간 간격 유지는 잘해주셨는데, StringCalculateTest에서는 간격 유지가 안되어 있네요 :)
 ```suggestion
    }
    
    @Test
```